MANIFEST.in

```diff
@@ -0,0 +1 @@
+include requirements.txt
```

Comment:
I dont't think this is needed. What is the value of packaging requirements.txt?

Reply:
oh, this is so dependencies can be computed as part of installing from pip.
It may just be easier to add it directly inside the setup.py .... it is only 1 dependency :) `six` and get rid of `requirements.txt` altogether.

Reply:
yes, but consider a case when we go more in the functionality of this package. In that situation, if we are not doing this approach then we have to manually add deps in `requirements.txt` as well as in `setup.py`.
Although this is a more favourable workflow which is used in every python package now. :)

Reply:
> if we are not doing this approach then we have to manually add deps in requirements.txt as well as in setup.py

I am suggesting to get rid of requirements.txt altogether and just maintain the `install_requires` list in setup.py. I don't foresee a huge amount of dependencies being added here. having everything in a single file is much easier.